pkg/skyenv/values.go

```diff
@@ -31,7 +31,7 @@ const (
 	DefaultUptimeTrackerAddr   = "http://ut.skywire.skycoin.com"
 	DefaultAddressResolverAddr = "http://ar.skywire.skycoin.com"
 	DefaultSetupPK             = "0324579f003e6b4048bae2def4365e634d8e0e3054a20fc7af49daf2a179658557"
-	DefaultNetworkMonitorPK    = ""
+	DefaultNetworkMonitorPKs   = ""
```

Comment:
Since it contains a single PK `DefaultNetworkMonitorPK` is correct.

Reply:
It could be multiple PKs later.